robotoff/insights/extraction.py

```diff
@@ -118,6 +118,7 @@ def extract_ocr_predictions(
 
 def extract_nutriscore_label(
```

Comment:
you have to add this parameter at the line where this function is called, and you have maybe to add this parameter to this function as well, and then to the place where we call it !

Reply:
Do you mean pass `source_image` as a parameter everywhere we call `extract_nutriscore_label()` in Robotoff?

Reply:
yes, it's just a few line above.